openfisca_us/parameters/demographic/geography/population_by_county.yaml

```diff
@@ -0,0 +1,65 @@
+description: Population by State and county name.
+metadata:
+  unit: person
+  reference:
+    - title: County Population Totals | Census.gov
+      href: https://www.census.gov/data/tables/time-series/demo/popest/2020s-counties-total.html
+UT:
```

Comment:
So we'd add populations either here or in the three-digit ZIP for every state we add?

Could you add Washington and Maryland while you're in here, as those are our next completed states?

Reply:
Removing these calibration parameters for now because we can generate them later from the new dataset if needed